src/en/space-station-14/character-species/guidelines.md

```diff
@@ -1,5 +1,238 @@
+
+For example, if an aquatic species has to live inside a water-filled suit at all times, they should still have the ability to wear a variety of outfits on top of that suit so they can express themselves like every other species. 
+
+This rule also covers other forms of rule-breaking. For example, a species could be themed around demons, but specifically theming a species around certain succubus/incubus tropes would obviously be too problematic to be suitable for Wizard's Den.
+
+### Rule 5: Deliberately Challenging Species
+
+Species **must** not be designed in such a way that a new player, picking a random character and rolling that species, would have an excessively negative experience due to lack of game knowledge. This **does not** mean a species cannot have a notable, potentially deadly, downside. However, that downside **should** be recoverable from, presented obviously to a player who has no prior context of that species, and teachable within the game by other players.
+
+At a minimum, when picking a species which could be challenging to a new player, a guidebook entry exists to help that player play the species, and that guidebook entry **must** appear when that species is picked for the first time, and this page **must** be focused on assisting the player with how to play the species, not simply being relegated to an encyclopedia entry on that species's mechanics.
+
+For example: a species suffers from oxygen poisoning. This means they must wear internals at all times, except in specific circumstances where oxygen is not present. This species **should not** punish the player by dying rapidly if their internals fail or are removed, and, if they do die, they **should** be revivable.
+
+If a species is significantly challenging to play as, compared to normal round-start species, the following rules apply:
+- It **must not** be able to be selected round-start by new players without clear warnings and tutorialization in place.
+- It **should not** be able to be picked if a random species is chosen for a midround ghost role. 
+- It **should** be given careful consideration on its impact on certain team antagonists, such as Nuclear Operatives, if that species is played as one of the team. (For example, a species that cannot wear shoes may be significantly disadvantaging to a Nuclear Operative team that can't provide that species with access to slip resistance othererwise.)
```

Comment:
```suggestion
- It **should** be given careful consideration on its impact on certain team antagonists, such as Nuclear Operatives, if that species is played as one of the team. (For example, a species that cannot wear shoes may be significantly disadvantaging to a Nuclear Operative team that can't provide that species with access to slip resistance otherwise.)
```